prog/main.go

```diff
@@ -227,6 +230,10 @@ func main() {
 	flag.StringVar(&flags.probe.weaveAddr, "probe.weave.addr", "127.0.0.1:6784", "IP address & port of the Weave router")
 	flag.StringVar(&flags.probe.weaveHostname, "probe.weave.hostname", app.DefaultHostname, "Hostname to lookup in WeaveDNS")
 
+	// DNS Snooper
+	flag.BoolVar(&flags.probe.dnsSnooperEnabled, "probe.dns-snooper.enabled", true, "Enable DNS Snooper (to identify external services being contacted)")
+	flag.StringVar(&flags.probe.dnsSnooperInterface, "probe.dns-snooper.interface", "any", "Networking interface on which to snoop")
+
```

Comment:
*both* the "enabled" and the interface flags are overkill. Yes, we can think of reasons people may want them. But in practice I just don't think it's going to happen. And if it does then we can always add the configurability then.